### PR TITLE
FEATURE: QA-9454 Customize embedded insight on SDK dashboard (TestCafe)

### PR DESCRIPTION
**Pull request by mluu3**
- add 2 buttons change legend and change separator to customize for visualization insight and bucket component insight.
- add new identifier for column chart 
- add new test for chart configuration

### CODE REVIEW

**Comment by mluu3** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-925.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3312/ : FAILURE in 1m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4105/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3313/ : FAILURE in 1m 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4107/ : SUCCESS in 7s

**Comment by mluu3** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-925.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1121/ : FAILURE in 30m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4108/ : SUCCESS in 6s

**Comment by mluu3** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-925.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1122/ : FAILURE in 28m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4109/ : SUCCESS in 6s

**Comment by mluu3** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-925.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3330/ : FAILURE in 6m 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4126/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1128/ : FAILURE in 29m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4127/ : SUCCESS in 9s

**Comment by mluu3** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-925.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3384/ : FAILURE in 2m 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4190/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1153/ : SUCCESS in 35m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4191/ : SUCCESS in 12s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3417/ : FAILURE in 1m 45s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4230/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3421/ : FAILURE in 1m 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4237/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3422/ : FAILURE in 1m 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4239/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3427/ : FAILURE in 2m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4244/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by mluu3** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-925.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3430/ : SUCCESS in 10m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4247/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1171/ : SUCCESS in 36m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4249/ : SUCCESS in 29s

File: examples/src/routes/ChartConfiguration.jsx [hunk 1]
```diff
@@ -4,18 +4,30 @@ import ExampleWithSource from "../components/utils/ExampleWithSource";
...
+import ConfigurationColumnChartExample from "../components/ConfigurationColumnChartExample";
+import ConfigurationColumnChartExampleSRC from "!raw-loader!../components/ConfigurationColumnChartExample"; // eslint-disable-line
 import PieChartColorMappingExample from "../components/PieChartColorMappingExample";
 import PieChartColorMappingExampleSRC from "!raw-loader!../components/PieChartColorMappingExample"; // eslint-disable-line
 
 export const BarChartDynamic = () => (
     <div>
         <div>
-            <h1>Bar Chart with dynamic colors example</h1>
+            <h1>Customize chart bucket compoent</h1>
```
Comment (xMort) [inline examples/src/routes/ChartConfiguration.jsx]: Typo: compoent > component

File: examples/test/ChartConfiguration_test.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        .expect(
+            barChart
+                .find(".highcharts-series-1 rect")
+                .nth(0)
+                .getAttribute("fill"),
+        )
+        .eql("rgb(168,194,86)");
+});
+
+test("should be able to change configuration of visialization chart and render them", async t => {
```
Comment (xMort) [inline examples/test/ChartConfiguration_test.js]: Typo: visialization > visualization

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3436/ : SUCCESS in 11m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4259/ : SUCCESS in 9s

**Comment by mluu3** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-925.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1176/ : SUCCESS in 33m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4260/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1830/ : UNSTABLE in 43m 28s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3439/ : FAILURE in 22m 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/936/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1001/ : SUCCESS in 39m 24s

**Comment by mluu3** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3444/ : SUCCESS in 3m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4270/ : SUCCESS in 17s

**Comment by xMort** [comment]
lgtm

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1835/ : SUCCESS in 24m 08s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3452/ : SUCCESS in 17m 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/938/ : SUCCESS in 29s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1004/ : SUCCESS in 33m 45s

**Pull request merged**